packages/paste-core/components/help-text/src/HelpText.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import {ValueOf} from '@twilio-paste/types';
+import type {ValueOf} from '@twilio-paste/types';
 import {Box} from '@twilio-paste/box';
-import {Flex} from '@twilio-paste/flex';
-import {Text, safelySpreadTextProps} from '@twilio-paste/text';
-import {TextColor} from '@twilio-paste/style-props';
+import type {TextProps} from '@twilio-paste/text';
```

Comment:
We're not using `Text` anymore are we? Can this be `BoxProps` take from `box`?

We might even then be able to remove the `@twilio-paste/text` peer dependency if it's not used anymore?

Reply:
True! I just changed replaced `TextProps` references to `BoxProps`, and removed dependencies to Text and Flex (since we removed both)